test/test.js

```diff
@@ -79,6 +79,14 @@ describe('photon()', function () {
     assertPathname(photonedUrl, '/example.com/image.png');
   });
 
+  it('should handle blob: URLs', function() {
+    var url = 'blob:http://example.com/ddd1d6b0-f31b-4937-ae9e-97f1d660cf71';
+    var photonedUrl = photon(url);
+
+    assertHostedOnPhoton( photonedUrl );
+    assertPathname(photonedUrl, '/http//example.com/ddd1d6b0-f31b-4937-ae9e-97f1d660cf71');
```

Comment:
Is there supposed to be a leading slash here before http?

Reply:
Yes, the `pathname` property of an URL always has the leasing slash.

Reply:
See the original tests that were failint in `wp-calypso`: https://github.com/Automattic/wp-calypso/pull/19317#issuecomment-341691738

Reply:
Thanks, just verified that on master too.